pkg/image/reference/reference.go

```diff
@@ -62,6 +63,19 @@ func Parse(spec string) (DockerImageReference, error) {
 	return ref, nil
 }
 
+// GetSpecDigestTag returns the image digest from a DockerImageReference string.
+// example '@sha:22204fa2...' in an image 'registry/ocp/release@sha:22204fa2...'
+func GetSpecDigestTag(spec string) (string, error) {
+	namedRef, err := reference.ParseNamed(spec)
+	if err != nil {
+		return "", err
+	}
+	if named, ok := namedRef.(reference.Canonical); ok {
```

Comment:
`reference.Digested`

Reply:
eh.. i reused that from here, btw : ) https://github.com/openshift/library-go/blob/master/pkg/image/reference/reference.go#L52  from Parse

Reply:
ok, I've updated that, thanks